packages/typo3-guides-cli/composer.json

```diff
@@ -11,7 +11,8 @@
     },
     "require": {
         "ext-dom": "*",
-        "ext-libxml": "*"
+        "ext-libxml": "*",
+        "t3docs/typo3-docs-theme": "dev-main"
```

Comment:
The cli package should not depend on a theme package (used mainly for frontend). Maybe we can move the Typo3VersionMapping enum somewhere else?

We run in problems for releases with the constraint? Maybe we can use "self.version" instead of "dev-main"?